clang/include/clang/Tooling/Transformer/SourceCode.h

```diff
@@ -91,6 +91,10 @@ StringRef getExtendedText(const T &Node, tok::TokenKind Next,
 llvm::Error validateEditRange(const CharSourceRange &Range,
                               const SourceManager &SM);
 
+/// Determines whether \p Range is one that can be read from.
+llvm::Error validateRange(const CharSourceRange &Range, const SourceManager &SM,
+                          bool AllowSystemHeaders);
```

Comment:
Please explain `AllowSystemHeaders` in the comments. May not be totally obvious.

Reply:
done